redis_cloud/README.md

```diff
@@ -0,0 +1,68 @@
+# Agent Check: Redis Cloud
+
+## Overview
+
+Redis is a versatile and fast data store that supports strings, hashes, lists, sets, streams, and more. It also offers programmability, extensibility, persistence, clustering, and high availability features, as well as Redis Stack for modern data models and processing engines. 
+
+The [Redis Cloud][1] integration is intended for use with Redis CLoud deployments of Redis software. It is not for use with Redis Enterprise installations, for that see the separate Redis Enterprise integration.
+
+This integration provides metrics for three critical aspects of a cluster; database, node, and shard, and makes it easier to detect problems before they become critical. For a full list see the 'Metrics' section.
```

Comment:
```suggestion 
This integration provides metrics for databases, nodes, and shards), making it easier to detect problems before they become critical. 
```

Reply:
we've modified this paragraph to look like the one for redis_enterprise, which was already approved.